docs/develop/api-reference/collection-ttl.md

```diff
@@ -0,0 +1,38 @@
+---
+sidebar_position: 4
+sidebar_label: CollectionTTL object
+title: CollectionTTL object API reference
+description: Learn how to interact with the API for dictionary collection data types in Momento Serverless Cache.
+slug: /develop/api-reference/collections/collectionttl
+---
+
+# CollectionTTL object API reference
+
+This object is passed in to API methods to say whether a TTL should be updated and if so, what the new TTL value should be.
+
+This tries to make the process a little less bewildering by providing named constructors and copiers for various situations.
```

Comment:
nit: maybe rather than "a little less bewildering" we say "as straightforward as possible" or "more straightforward" or "more intuitive"?